engine/src/flutter/shell/platform/darwin/macos/BUILD.gn

```diff
@@ -18,6 +18,31 @@ shell_gpu_configuration("macos_gpu_configuration") {
   enable_metal = true
 }
 
+config("config_macos") {
+  include_dirs = [ target_gen_dir ]
+  lib_dirs = mac_swift_lib_paths
+
+  # Allow use of @testable imports in debug builds and when tests are enabled.
+  if (enable_ios_unittests) {
+    swiftflags = [ "-enable-testing" ]
+  }
+}
+
+config("config_macos_test") {
+  configs = [ ":config_ios" ]
```

Comment:
Should this be `config_macos`?

Reply:
Yowch... how embarrassing! Yes! That's what I get for not adding a Swift XCTest in this patch. See below on why.